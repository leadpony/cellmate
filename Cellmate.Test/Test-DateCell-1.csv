2020/2/2
2020/3/3
2030/4/4