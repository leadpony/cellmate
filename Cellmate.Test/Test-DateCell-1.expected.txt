Test-DateCell-1.csv:Test-DateCell-1:A1 2020/02/02 0:00:00
Test-DateCell-1.csv:Test-DateCell-1:A2 2020/03/03 0:00:00
Test-DateCell-1.csv:Test-DateCell-1:A3 2030/04/04 0:00:00